rfcs/text/0025-container-metric-fields.md

```diff
@@ -88,6 +89,246 @@ Stage 2: Included a real world example source document. Ideally this example com
+      "availableBytes": 1768316928,
+      "usageBytes": 2764943360,
+      "workingSetBytes": 2111090688,
+      "rssBytes": 2150400,
+      "pageFaults": 131567,
+      "majorPageFaults": 103
+    },
+    "network": {
+      "time": "2021-12-20T09:53:30.156Z",
+      "rxBytes": 1115133198,
+      "rxErrors": 0,
+      "txBytes": 812729002,
+      "txErrors": 0
+    },
+    "fs": {
+      "availableBytes": 98727014400,
+      "capacityBytes": 101258067968,
+      "usedBytes": 2514276352,
+      "inodesFree": 18446744073709551615,
+      "inodes": 6258720,
+      "inodesUsed": 138624
+    },
+    "runtime": {
+      "imageFs": {
+        "availableBytes": 98727014400,
+        "capacityBytes": 101258067968,
+        "usedBytes": 860204379,
+        "inodesFree": 18446744073709551615,
+        "inodes": 6258720,
+        "inodesUsed": 138624
+      }
+    }
+  },
+  "pods": [
+    {
+      "podRef": {
+        "name": "nginx-deployment-2303442956-pcqfc",
+        "namespace": "default",
+        "uid": "beabc196-2456-11e7-a3ad-42010a840235"
+      },
+      "startTime": "2021-12-20T09:53:30.156Z",
+      "containers": [
+        {
+          "name": "nginx",
+          "startTime": "2017-04-18T16:47:44Z",
+          "cpu": {
+            "time": "2017-04-20T08:06:34Z",
+            "usageNanoCores": 11263994,
+            "usageCoreNanoSeconds": 43959424
+          },
+          "memory": {
+            "time": "2017-04-20T08:06:34Z",
+            "usageBytes": 1462272,
+            "workingSetBytes": 1454080,
+            "rssBytes": 1409024,
+            "pageFaults": 841,
+            "majorPageFaults": 0
+          },
+          "rootfs": {
+            "availableBytes": 98727014400,
+            "capacityBytes": 101258067968,
+            "usedBytes": 61440,
+            "inodesFree": 6120096,
+            "inodes": 6258720,
+            "inodesUsed": 21
+          },
+          "logs": {
+            "availableBytes": 98727014400,
+            "capacityBytes": 101258067968,
+            "usedBytes": 28672,
+            "inodesFree": 6120096,
+            "inodes": 6258720,
+            "inodesUsed": 138624
+          },
+          "userDefinedMetrics": null
+        }
+      ],
+      "network": {
+        "time": "2021-12-20T09:53:30.156Z",
+        "rxBytes": 107056,
+        "rxErrors": 0,
+        "txBytes": 72447,
+        "txErrors": 0
+      },
+      "volume": [
+        {
+          "availableBytes": 1939689472,
+          "capacityBytes": 1939701760,
+          "usedBytes": 12288,
+          "inodesFree": 473551,
+          "inodes": 473560,
+          "inodesUsed": 9,
+          "name": "default-token-sg8x5"
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### Mapped event
+
+```json
+{
+  "container": {
+    "cpu.usage":          0.005631997,
+    "memory.usage":       0.01,
+    "name": "nginx",
+    "network.ingress.bytes": 107056,
+    "network.egress.bytes": 72447
+  },
+  "kubernetes": {
```

Comment:
Unrelated to the focus here, but ECS does have [`orchestrator.*`](https://www.elastic.co/guide/en/ecs/current/ecs-orchestrator.html) fields which might fit some of the k8s fields.